src/node_messaging.cc

```diff
@@ -1645,6 +1645,31 @@ static void BroadcastChannel(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void ExposeLazyDOMExceptionPropertyGetter(
+    v8::Local<v8::Name> name, const v8::PropertyCallbackInfo<v8::Value>& info) {
+  auto context = info.GetIsolate()->GetCurrentContext();
+  Local<Function> domexception = GetDOMException(context).ToLocalChecked();
+  info.GetReturnValue().Set(domexception);
```

Comment:
```suggestion
  Local<Function> domexception;
  if (GetDOMException(context).ToLocal(&domexception)) {
    info.GetReturnValue().Set(domexception);
  }
```

Reply:
FWIW I copied that from https://github.com/nodejs/node/blob/f0cb5d4e6f00b83b4e75bec103239b0ba0a1a592/src/node_messaging.cc#L1722-L1727

Which makes me think, maybe it would make more sense to get that value back instead of creating a new instance